content/blog/2024-10-09-Azure-Marketplace-Updates.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Azure Marketplace Updates"
+type: blog
+author: 
+ name: "Elkhan Mammadli"
+ bio: "AlmaLinux OS Cloud engineer"
+ image: /users/elkhan.jpg
+date: '2024-10-09'
+post:
+    title: "Azure Marketplace Updates"
+    image: /blog-images/2024/azure-marketplace-updates.png
+
+---
+
+Hello Community! We'd like to share a quick update for our AlmaLinux Azure users. 
+
+If you've been using the original AlmaLinux Azure image for **x86_64** architecture (VM Image URN: `almalinux:almalinux:*`), you may have gotten an email like this:
+
+> Your Workloads are running on images that will be deprecated soon
+> 
+> You're receiving this email because you have virtual machine and/or virtual machine scale set deployments that are running on images scheduled for deprecation.
+> 
+> One or more offers from the publisher, almalinux, have been scheduled for deprecation. You currently have workloads that are running on images within these offers.
+> 
+> Impact
+> 
+> • Following the scheduled deprecation date, you won't be able to deploy any additional virtual machine (VM) or virtual machine scale set (VMSS) instances using any images within these offers.
+>
+> • Your active VM instances won't be affected following the deprecation date.
+>
+> • Your existing VMSS deployments will continue to be operational but can't be scaled out following deprecation.
```

Comment:
```suggestion
> _Your Workloads are running on images that will be deprecated soon_
> 
> _You're receiving this email because you have virtual machine and/or virtual machine scale set deployments that are running on images scheduled for deprecation._
> 
> _One or more offers from the publisher, almalinux, have been scheduled for deprecation. You currently have workloads that are running on images within these offers._
> 
> **_Impact_**
> 
> • _Following the scheduled deprecation date, you won't be able to deploy any additional virtual machine (VM) or virtual machine scale set (VMSS) instances using any images within these offers._
>
> • _Your active VM instances won't be affected following the deprecation date._
>
> • _Your existing VMSS deployments will continue to be operational but can't be scaled out following deprecation._
```

Reply:
It looks like the decoration for blockquote currently only applies to` .al-page-index .al-index-press .al-index-press-item` so we should look at making that apply universally in the future. I don't love it, but for now we can just add italics to the text manually.